containers/ws/label-run

```diff
@@ -42,8 +42,10 @@ else
 
     /usr/libexec/cockpit-certificate-ensure
 
-    if [ -z "$SSH_AUTH_SOCK" ]; then
+    if [ -z ${SSH_AUTH_SOCK+x} ]; then
         eval "$(ssh-agent)"
+    else
+        echo Agent sock ${SSH_AUTH_SOCK}
```

Comment:
Debugging leftover, please drop.

Reply:
Actually I put that in on purpose, as the previous output was `Agent pid N ` .
But sure, can remove.

Reply:
I meant it's not really useful as that path is only valid from inside the container?

Reply:
I meant, it's not really  useful, as the user explicitly specified the path -- there's no surprise?